halo2-ecc/src/fields/vector.rs

```diff
@@ -344,6 +344,18 @@ where
             self.fp_chip.assert_equal(ctx, a_coeff, b_coeff)
         }
     }
+
+    pub fn select_or_zero<A>(
```

Comment:
Add comments

Reply:
In general rust docs only show up for the trait definition, but I will add it for dev convenience.